pontoon/administration/templates/admin_project_strings.html

```diff
@@ -9,16 +9,13 @@
 
   <h1>
     {{ 'Strings for %(project)s' | format(project=project.name) }}
-    <aside>
-      <a href="{{ url('pontoon.admin.project.strings', project.slug) }}?format=csv">Download Strings</a>
-    </aside>
```

Comment:
I instinctively disagree with removing this, but I'm sure you have excellent reasons. Please enlighten me! :smile:

Reply:
I don't have better arguments than what the commit message says. :) https://github.com/adngdb/pontoon/pull/1/commits/5db4d5f946e4d750eb289e49b59ae29d756dda51